### PR TITLE
PMM-5988 Add fields TotalSteps and FinishedSteps

### PR DESCRIPTION
**Pull request by jprukner**
[PMM-5988](https://jira.percona.com/browse/PMM-5988)

- [x] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1423

- [x] Links to other linked pull requests (optional).
  - https://github.com/percona-platform/dbaas-controller/pull/129
  - https://github.com/percona/pmm/pull/656
  - https://github.com/percona-platform/dbaas-api/pull/85 
---
- [x] Tests passed.
- [ ] Feature build pass.
- [ ] (Re)requested review.
- [ ] Fix conflicts with target branch.
- [x] Update API dependency.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

### CODE REVIEW

**Review by askomorokhov — APPROVED** [review]

File: services/management/dbaas/psmdb_cluster_service.go [hunk 1]
```diff
@@ -89,7 +89,9 @@ func (s PSMDBClusterService) ListPSMDBClusters(ctx context.Context, req *dbaasv1
 			},
 			State: psmdbStates()[c.State],
 			Operation: &dbaasv1beta1.RunningOperation{
-				Message: message,
+				TotalSteps:    c.Operation.TotalSteps,
+				FinishedSteps: c.Operation.FinishedSteps,
+				Message:       message,
```
Comment (BupycHuk) [inline services/management/dbaas/psmdb_cluster_service.go:94]: let's remove https://github.com/percona/pmm-managed/pull/623/files#diff-7a56b03a49f9c4fd843c3edfc194110f786609bba47b3ec694b844ec0f3aff8cR78 check and use `c.Operation.Message` directly

**Review by BupycHuk — APPROVED** [review]
only one minor comment

File: services/management/dbaas/psmdb_cluster_service.go (hunk 1 above)
Comment (jprukner) [reply services/management/dbaas/psmdb_cluster_service.go]: @BupycHuk I removed the check. Did you mean it like that?

Comment (BupycHuk) [reply services/management/dbaas/psmdb_cluster_service.go]: yes, thank you

**Pull request merged**